manifests/install.pp

```diff
@@ -40,9 +40,9 @@
 
   if !$::vault::disable_mlock {
     exec { "setcap cap_ipc_lock=+ep ${vault_bin}":
```

Comment:
Agree that it's a nicer alternative, I've opened #50 to track the issue.

@cruwe If you just want to get the functionality working in this PR I'm happy to merge as-is (once tests are passing), but you're also free to tackle #50 if you feel like it.